package.json

```diff
@@ -10,7 +10,8 @@
     "lint": "eslint ./src/Frontend && lessc src/Frontend/styles/custom.less --lint",
     "prod": "NODE_ENV=production npm run frontend",
     "test": "mocha src/Frontend/test --recursive --ui bdd --compilers js:babel-core/register --reporter dot --require babel-polyfill --require src/Frontend/test/DOM.js",
-    "watch": "webpack --watch"
+    "watch": "webpack --watch",
+    "postinstall": "touch ./src/Frontend/config/recaptcha.js"
```

Comment:
it shouldn't be created if it exists already and to the new file an 'export default ""' should be added. Would recommend to move it into a custom shell in bin/